CodeEdit/Features/NavigatorArea/OutlineView/FileSystemTableViewCell.swift

```diff
@@ -40,7 +40,52 @@ class FileSystemTableViewCell: StandardTableViewCell {
         fileItem = item
         imageView?.image = item.nsIcon
         imageView?.contentTintColor = color(for: item)
-        textField?.stringValue = item.labelFileName()
+
+        let fileName = item.labelFileName()
+
+        // Apply bold style if the filename matches the workspace filter
+        if let filter = workspace?.filter, !filter.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty {
```

Comment:
This cell class is used outside of the project navigator. Instead of grabbing the filter from the workspace, lets add an optional filter property and set it from the project navigator. That'll prevent the filter style from leaking into other UI elements like the search list.